.github/workflows/pr-author-assigne.yml

```
@@ -0,0 +1,105 @@
+permissions:
+  pull-requests: write
+  issues: read
+
+jobs:
+  validate-pr:
+    runs-on: ubuntu-24.04-arm
+
+    steps:
+      - name: Validate linked issue
+        id: validate
+        run: |
```

actionlint (1.7.8)

"github.event.pull_request.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)